crates/runtime/Cargo.toml

```diff
@@ -130,6 +130,8 @@ async-graphql-axum = "7.0.5"
 bollard = "0.16.1"
 spice-cloud = { path = "../spice_cloud" }
 tracing-subscriber.workspace = true
+tracing-opentelemetry.workspace = true
+opentelemetry_sdk = { workspace = true, features = ["rt-tokio-current-thread"] }
```

Comment:
Feature is required for [TokioCurrentThread](https://docs.rs/opentelemetry_sdk/0.24.1/opentelemetry_sdk/runtime/struct.TokioCurrentThread.html#). With Tokio runtime tests hang similar to https://github.com/open-telemetry/opentelemetry-rust/issues/868